app/controllers/guide_parts_controller.rb

```diff
@@ -13,36 +13,51 @@ class GuidePartsController < InheritedResources::Base
   before_action do
     require_user_accessibility_to_edition(@edition)
   end
-  before_action only: %i[new create] do
+  before_action only: %i[new create edit update] do
     require_editor_permissions(@edition)
   end
-  before_action only: %i[new create] do
+  before_action only: %i[new create edit update] do
     require_editing_state(@edition)
   end
 
-  def new
-    @part = Part.new
-
-    render "secondary_nav_tabs/guide_add_new_chapter_page"
-  end
-
   def create
     @part = @edition.editionable.parts.build(permitted_parts_params.merge(order: @edition.editionable.parts.size))
     if @edition.save
       if params[:save] == "save and summary"
         flash[:success] = "New chapter added successfully."
         redirect_to edition_path(@edition)
       elsif params[:save] == "save"
-        flash[:success] = "Not implemented yet."
+        flash[:success] = "New chapter added successfully."
+        redirect_to edit_edition_guide_part_path(@edition, @part)
+      end
+    else
+      render "new"
+    end
+  rescue StandardError => e
+    Rails.logger.error "Error #{e.class} #{e.message}"
+    @edition.errors.add(:show, "Due to a service problem, the edition couldn't be updated")
+    render "new"
+  end
+
+  def update
```

Comment:
We need to deal with permissions when the user clicks "Save" or "Save and go to summary" (see AC4). If the user is on this page but does not have editor permissions then they should be taken to the "Edit" page with an error message.